server/fleet/app.go

```diff
@@ -120,9 +120,24 @@ type VulnerabilitySettings struct {
 	DatabasesPath string `json:"databases_path"`
 }
 
+// MDMAppleABMAssignmentInfo represents an user definition of the association
+// between an ABM token (via organization name) and the teams used to associate
+// hosts when they're ingested during the ABM sync.
+type MDMAppleABMAssignmentInfo struct {
```

Comment:
Is the organization name unique? I don't recall how Apple handles this. 

If not, do we care? Or this just being used for serialization purposes?

Reply:
@gillespi314 IIRC the org name is not unique on Apple's side, but we're treating it as unique